portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const quotes =
+      ['We cannot change the cards we are dealt, just how we play the hand.', 
+      'When it comes to men who are romantically interested in you, it’s really simple. Just ignore everything they say and only pay attention to what they do.', 
+      'The key question to keep asking is, Are you spending your time on the right things? Because time is all you have.', 
+      'Showing gratitude is one of the simplest yet most powerful things humans can do for each other.'];
+    
+  // Pick a random quote.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
+}
+
+/* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
+function onNavClick() {
+  var x = document.getElementById("my-topnav");
+  if (x == null) {
+      return // no-op
+  }
+  if (x.className === "topnav") {
+    x.className += " responsive";
+  } else {
+    x.className = "topnav";
+  }
+}
+
+// Automatic Slideshow - change image every 3 seconds
+var myIndex = 0;
+
+function carousel() {
+    var i;
```

Comment:
Don't expose the scope if it's not needed. E.g, code outside the for loop should not be able to know about this `i`

Reply:
fixed thank you